.gitignore

```diff
@@ -4,5 +4,5 @@ npm-debug.log
 .env
 package-lock.json
 coverage
-lib
+dist
```

Comment:
It's a moot point now, but I was thinking about this earlier. In cases like this, should we go with default configuration, or make changes base on our own arbitrary, not-so-conventional conventions? 

So to apply that to this case, I'd ask, "in our industry, is it a greater-used convention to use `lib` or `dist`?"

How would we even go about answering that question? If we can't, then should we adopt our conventions, or adopt whatever defaults we're given?

Reply:
1. Be consistent
2. If in doubt, refer to the first rule

Ideally `being consistent === the defaults`.

I think defaults are what we should strive for. But we should also accept that sometimes we have deviated, and shouldn't start Yet-Another-Migration by trying to move back to the defaults half-heartedly.